README.md

```diff
@@ -1 +1,16 @@
-# hypertrace-gradle-code-style-plugin
+# Hypertrace Code Style Plugin
+
+###### org.hypertrace.code-style-plugin
+
+[![CircleCI](https://circleci.com/gh/hypertrace/hypertrace-gradle-code-style-plugin.svg?style=svg)](https://circleci.com/gh/hypertrace/hypertrace-gradle-code-style-plugin)
```

Comment:
this should be the GH actions badge (other repos may be out of date)

Reply:
yup. or we can remove this as of now. I removed this CCI badge from our commonly used repos but I guess we need to remove them from plugin repositories and template repository too.